src/main/java/castalia/Dist.java

```diff
@@ -0,0 +1,381 @@
+                0.065682337918207449113 };
+
+        double b[] = {
+                47.20258190468824187, 976.09855173777669322,
+                10260.932208618978205, 45507.789335026729956} ;
+
+        double one = 1.0;
+        double half = 0.5;
+        double zero = 0.0;
+        double sixten = 1.6;
+        double sqrpi = 1./SQRT2PI;
+        double thrsh = 0.66291;
+        double root32 = 5.6568542494923806;
+        final double DBL_EPSILON=2.2204460492503131e-16;
+        //final double DBL_MIN=2.2250738585072014e-308; ?
+
+        double xden, temp, xnum, result, ccum;
+        double del, min, eps, xsq;
+        double y;
+        int i;
+
+        eps = DBL_EPSILON * .5;
+        min = Double.MIN_VALUE; //DBL_MIN;
+        y = Math.abs(x);
+        if (y <= thrsh) {
+	/* Evaluate pnorm for |z| <= 0.66291 */
+            xsq = zero;
+            if (y > eps) { xsq = x * x; }
+            xnum = a[4] * xsq; xden = xsq;
+            for (i = 1; i <= 3; ++i)
+            {  xnum = (xnum + a[i - 1]) * xsq;
+                xden = (xden + b[i - 1]) * xsq;
+            }
+            result = x * (xnum + a[3]) / (xden + b[3]);
+            temp = result; result = half + temp;
+            ccum = half - temp;
+        }
+        else if (y <= root32) {
+    	/* Evaluate pnorm for 0.66291 <= |z| <= sqrt(32) */
+            xnum = c[8] * y; xden = y;
+            for (i = 1; i <= 7; ++i)
+            {  xnum = (xnum + c[i - 1]) * y;
+                xden = (xden + d[i - 1]) * y;
+            }
+            result = (xnum + c[7]) / (xden + d[7]);
+            xsq = fint(y * sixten) / sixten;
+            del = (y - xsq) * (y + xsq);
+            result = Math.exp(-xsq*xsq*half) * Math.exp(-del*half)*result;
+            ccum = one - result;
+            if (x > zero) { temp = result; result = ccum; ccum = temp; }
+        }
+        else {
+    	/* Evaluate pnorm for |z| > sqrt(32) */
+            result = zero; xsq = one / (x * x);
+            xnum = p[5] * xsq; xden = xsq;
+            for (i = 1; i <= 4; ++i)
+            {  xnum = (xnum + p[i - 1]) * xsq;
+                xden = (xden + q[i - 1]) * xsq;
+            }
+            result = xsq * (xnum + p[4]) / (xden + q[4]);
+            result = (sqrpi - result) / y;
+            xsq = fint(x * sixten) / sixten;
+            del = (x - xsq) * (x + xsq);
+            result = Math.exp(-xsq*xsq*half) * Math.exp(-del*half)*result;
+            ccum = one - result;
+            if (x > zero) { temp = result; result = ccum; ccum = temp; }
+        }
+        if (result < min) { result = 0.0; }
+        if (ccum < min) { ccum = 0.0; }
+        return result;
+    }
+
+    /** integer floor */
+    public static double fint(double x)
+    { return (x >= 0.0) ? Math.floor(x) : -Math.floor(-x); }
+
+
+    // From R source
+    /** standard normal quantile function */
+    public static double qnorms(double p)
+    { double q, r, val;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Use one line for each declaration, it enhances code readability.](https://www.codacy.com/app/scala-academy/Castalia/file/1968742700/issues/source?bid=2998213&fileBranchId=3112713#l131)